.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,6 +17,9 @@ If you can, provide a minimal chunk of code that shows the problem (either inlin
 Please attach the clangd stderr log if you can. (Usually available from the editor)
 If possible, run with `--log=verbose` - note that the logs will include the contents of open files!
 If this is a crash, try to put `llvm-symbolizer` on your PATH per the troubleshooting instructions.
+(If you're using Windows, place the associated PDB file(debug symbols) in the same directory as
```

Comment:
rather than requiring this extra step, it'd be nice just to include the pdb file in the right location in the windows clangd.zip

Do you think it is too large in practice?
I like having the binaries be reasonably compact

(Ideally I think we'd ship by default a tiny debuginfo that just had just enough to symbolize stacktraces - this is trivially small as `-gmlt` on other platforms shows - but don't want to ask you to dig into that if it's not really obvious)